client/app/telecom/telephony/billingAccount/administration/addGroup/telecom-telephony-billing-account-administration-add-group.html

```diff
@@ -1,66 +1,56 @@
 <section>
-
     <header>
-        <section-back-link section-back-link-to-state="telecom.telephony.administration"
-                           section-back-link-title="{{:: 'telephony_group_admin_group_back_link' | translate }}">
-        </section-back-link>
-        <h1 data-translate="telephony_add_group_title"></h1>
+        <oui-back-button
+            data-title="{{:: 'telephony_add_group_title' | translate }}"
+            data-on-click="AddGroupCtrl.back()">
```

Comment:
Why a `button` is considered as a link ?
It should be a `back-link` with a given `ui-state` instead of a `back-button` with a `ng-click`.

Reply:
Okay. It is not possible to consider it as a link because we didn't want to put ui-router as a dependency.

Reply:
It's always possible ;-) As all manager (and almost all our apps) are using ui-router as dependency, I don't think it's a blocking point. But...
Another solution could be to give an `href` using `$state.href()` (see https://ui-router.github.io/ng1/docs/0.3.1/index.html#/api/ui.router.state.$state#methods_href). Like this it can be a link and be more compliant with accessibility and HTML standards :-)